src/site/stages/build/index.js

```diff
@@ -0,0 +1,166 @@
+// Builds the site using Metalsmith as the top-level build runner.
+const Metalsmith = require('metalsmith');
+const assets = require('metalsmith-assets');
+const collections = require('metalsmith-collections');
+const dateInFilename = require('metalsmith-date-in-filename');
+const filenames = require('metalsmith-filenames');
+const inPlace = require('metalsmith-in-place');
+const layouts = require('metalsmith-layouts');
+const liquid = require('tinyliquid');
+const markdown = require('metalsmith-markdownit');
+const moment = require('moment');
+const navigation = require('metalsmith-navigation');
+const permalinks = require('metalsmith-permalinks');
+
+const getOptions = require('./options');
+const createBuildSettings = require('./middleware/create-build-settings');
```

Comment:
Is middleware the right word here? These are technically Metalsmith plugins.

Reply:
Ah yep, `'plugin` is the right word. I've been saying "middleware" for ages 😆